kcidb/test_oo.py

```diff
@@ -0,0 +1,192 @@
+"""kcidb.oo test module"""
+
+import kcidb
+
+
+class KCIDBOOClientTestCase(kcidb.unittest.TestCase):
+    """Test case for oo.Client"""
+
+    def check_type_instance(self, item, type_name, type_class):
+        """Check the instance of a type name."""
+        for i in item[type_name]:
+            self.assertIsInstance(i, type_class)
+
+    def check_data(self, obj):
+        """Run `check_type_instance` on each type."""
+        for type_name in obj:
+            if type_name == 'revision':
+                self.check_type_instance(obj, type_name, kcidb.oo.Revision)
+            elif type_name == 'checkout':
+                self.check_type_instance(obj, type_name, kcidb.oo.Checkout)
+            elif type_name == 'build':
+                self.check_type_instance(obj, type_name, kcidb.oo.Build)
+            else:
+                self.check_type_instance(obj, type_name, kcidb.oo.Test)
```

Comment:
Hmm, why don't you use the kcidb.oo.CLASSES dictionary?

Reply:
I've updated it. `kcidb.oo.CLASSES` is a no-brainer.